assets/js/script.js

```diff
@@ -119,209 +102,214 @@ var randomQuestion;
+    if (nextBtn.classList.contains('hide')) {
+      nextBtn.classList.toggle('hide');
+    }
+    if (parseInt(answer) === questions[currentQuestion].correctAnswer) {
+      localStorage.setItem('selectedAnswer', 'right');
+    } else {
+      localStorage.setItem('selectedAnswer', 'wrong');
     }
+  }
 });
 
-nextBtn.addEventListener('click', function() {
-    var getStorage = localStorage.getItem("selectedAnswer");
-    
-    if (getStorage === "right") {
-        console.log("y");
-        score += 10;
-        scoreBoard.textContent = score;
-    } 
-    else {
-        console.log("n");
-        timerCount -= 5;
-        secondsLeft.textContent = timerCount;
-    };
-        setTimeout(function() {
-        result.textContent = score + "%";
-        localStorage.removeItem("selectedAnswer");
-        setNextQuestion(currentQuestion++);
-    },600);
+nextBtn.addEventListener('click', function () {
+  var getStorage = localStorage.getItem('selectedAnswer');
+
+  if (getStorage === 'right') {
+    console.log('y');
+    score += 10;
+    scoreBoard.textContent = score;
+    correctSound.play();
+  } else {
+    console.log('n');
+    timerCount -= 5;
+    secondsLeft.textContent = timerCount;
+    wrongSound.play();
+  }
+  setTimeout(function () {
+    result.textContent = score + '%';
+    localStorage.removeItem('selectedAnswer');
+    setNextQuestion(currentQuestion++);
+  }, 600);
 });
 
 // results page
-var result = document.querySelector(".result");
-var userInput = document.querySelector(".name-input"); //grabs input field
-var submitBtn = document.querySelector(".submit"); // grabs submit button
-var highScoreForm = document.getElementById("high-score-input"); //grabs entire form
-var highScoreList = document.getElementById("high-score-list"); //grabs empty ul
-var resultsMsg = document.querySelector(".results-msg");
-var highScoresArray = JSON.parse(localStorage.getItem("High Scores"))|| [] ; //creates empty array for each score to be added to
+var result = document.querySelector('.result');
+var userInput = document.querySelector('.name-input'); //grabs input field
+var submitBtn = document.querySelector('.submit'); // grabs submit button
+var highScoreForm = document.getElementById('high-score-input'); //grabs entire form
+var highScoreList = document.getElementById('high-score-list'); //grabs empty ul
+var resultsMsg = document.querySelector('.results-msg');
+var highScoresArray = JSON.parse(localStorage.getItem('High Scores')) || []; //creates empty array for each score to be added to
 
 function gameEndMessage() {
-    if (score === 100) {
-        resultsMsg.textContent = "Perfect Score!";
-    }
-    if (score >= 70 && score < 100) {
-        resultsMsg.textContent = "Great Work!";
-    }
-    if (score >= 50 && score < 70) {
-        resultsMsg.textContent = "Keep It Up!"
-    }
-    if (score < 50) {
-        resultsMsg.textContent = "Better Luck Next Time"
-    }
+  if (score === 100) {
```

Comment:
Check out `switch/case` statements (if supported in JS) for handling blocks of `if`s like this - also notice that there isn't a case here where one of the statements is going to be true and the the others will also be true, so it's really an `if/else` rather than a series of `if`s